server/odc-core/src/main/java/com/oceanbase/odc/core/sql/util/FullLinkTraceUtil.java

```diff
@@ -48,7 +48,12 @@ public static SqlExecTime getFullLinkTraceDetail(Statement statement) throws SQL
                 throw new UnexpectedException("No trace info, maybe value of ob_enable_show_trace is 0.");
             }
             String showTraceJson = resultSet.getString(1);
-            SqlExecTime execDetail = parseSpanList(JsonUtils.fromJsonList(showTraceJson, TraceSpan.class));
+            SqlExecTime execDetail;
+            try {
+                execDetail = parseSpanList(JsonUtils.fromJsonList(showTraceJson, TraceSpan.class));
+            } catch (Exception e) {
+                throw new UnexpectedException("Parse trace failed, original text: " + showTraceJson, e);
```

Comment:
showTraceJson will be too large, don't print it directly

Reply:
done.